charts/selenium-grid/index.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
why do we need this file?

Reply:
when adding helm repo, it look for the index file. The index is generate using helm command
`helm repo index selenium-grid/ --url <github-repo-of-chart>